source/installation-guide/installing-elastic-stack/elastic_server_rpm.rst

```diff
@@ -39,7 +39,22 @@ Elasticsearch is a highly scalable full-text search and analytics engine. For mo
 
     # yum install elasticsearch-7.3.0
 
-2. Enable and start the Elasticsearch service:
+2. **Optional.** Elasticsearch will only listen on the loopback interface (localhost) by default. Configure Elasticsearch to listen on all interfaces by editing the file ``/etc/elasticsearch/elasticsearch.yml`` and uncommenting the setting ``network.host``. Change the value to:
+
+   .. code-block:: yaml
+
+     network.host: 0.0.0.0
+
+3. If you are installing a **distributed architecture**, you will have to make an additional configuration change by editing the file ``/etc/elasticsearch/elasticsearch.yml``. Add or edit (if commented) the following lines:
+
+   .. code-block:: yaml
+
+     node.master: true
```

Comment:
This is the default value so we may want to remove this line (see [modules-node](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-node.html)).

Reply:
Removed in this [commit](https://github.com/wazuh/wazuh-documentation/pull/1446/commits/c09621c8f7ee49e368621a925aa3ae5b3ae4c96d). Thanks.